get_CustomDevices.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+Import("env")
+
+CUSTOMDEVICES_DIR = env.subst("$PROJECT_DIR/CustomDevices")
+
+if not os.path.exists(CUSTOMDEVICES_DIR):
+    print ("Cloning Mobiflight-CustomDevices repo ... ")
+    env.Execute("git clone --depth 100 https://github.com/elral/MobiFlight-CustomDevices $PROJECT_DIR/CustomDevices")
+else:
+    print ("Checking for Mobiflight-CustomDevices repo updates ... ")
+    env.Execute("git --work-tree=$PROJECT_DIR\CustomDevices --git-dir=$PROJECT_DIR\CustomDevices\.git pull origin main --depth 100")
```

Comment:
This file gets executed on each build from an environment. It clones the repo for custom devices or checks for updates

Reply:
Why is this script needed? The repo already gets cloned by the changes made to the GitHub workflow scripts.

Reply:
OK, turns out I asked the same question (in a complementary way) before reading this... 
The way I see it, I'm inclined to say that the cloning here should remain - for local builds it has to be there somewhere, For automated builds it can be in the script if this instance isn't applicable to that case for whatever reason (which is my question elsewhere).

Reply:
Also as it stands now this is causing an error in the automated build process:

```
git --work-tree=/home/runner/work/MobiFlight-FirmwareSource/MobiFlight-FirmwareSource\CustomDevices --git-dir=/home/runner/work/MobiFlight-FirmwareSource/MobiFlight-FirmwareSource\CustomDevices\.git pull origin main --depth 100
fatal: not a git repository: '/home/runner/work/MobiFlight-FirmwareSource/MobiFlight-FirmwareSourceCustomDevices.git'
```

Reply:
This was due to using backslashes in the path.